ccpp/data/GFS_typedefs.meta

```diff
@@ -6651,6 +6657,18 @@
   units = flag
   dimensions = ()
   type = logical
+[levozp]
+  standard_name = number_of_levels_in_ozone_data
```

Comment:
Why not reuse `vertical_dimension_of_ozone_forcing_data`?

Reply:
Not sure why I did this. I will revert.